packages/calling-stateful-client/src/Converter.ts

```diff
@@ -125,6 +125,9 @@ export function convertSdkCallToDeclarativeCall(call: CallCommon): CallState {
       currentCaptionLanguage: '',
       currentSpokenLanguage: '',
       isCaptionsFeatureActive: false
+    },
+    optimalVideoCountFeature: {
+      optimalVideoCount: 1
```

Comment:
when a call first comes in, is there already a known optimalVideoCount? e.g. could this be:
```suggestion
      optimalVideoCount: call.feature(Features.OptimalVideoCount).optimalVideoCount
```
?

Reply:
Coming from the SDK if the call is initializing the default value at the beginning will be 1 as well

Reply:
Is there a reason to not do `optimalVideoCount: call.feature(Features.OptimalVideoCount).optimalVideoCount`? Just in case this implementation detail of it starting as `1` changes in future?

Reply:
just analyzed this comment and addressed as suggested